### ISSUE TITLE
Is Release 1.7.6 tested under terraform 0.13.x ? 

### ISSUE THREAD

**Issue by syyang-in-cloud** [scottwinkler/terraform-provider-shell#84]
I'm getting an error on running `update.sh` that there is no StdIn when I am running with terraform 0.13.5:
```
TestPluginsProviderShell 2020-11-17T23:00:49Z logger.go:66: StdIn: 
TestPluginsProviderShell 2020-11-17T23:00:49Z logger.go:66: '{}'
TestPluginsProviderShell 2020-11-17T23:00:49Z logger.go:66: 
TestPluginsProviderShell 2020-11-17T23:00:49Z logger.go:66: 
```
`create/read/delete` seems working fine, but `update` is getting no StdIn.

Have not tested in 0.12.x whether it's working for update or not.

**Comment by scottwinkler** [comment]
Try 1.7.4. There were some changes to update in 1.7.6 which might be causing problems

**Comment by syyang-in-cloud** [comment]
> Try 1.7.4. There were some changes to update in 1.7.6 which might be causing problems

Thanks for the quick reply, and yes, I pinned the version `>=1.7.4,<1.7.6`. Any reason not to try 1.7.5?

**Comment by scottwinkler** [comment]
1.7.5 and 1.7.6 are related

**Comment by rucciva** [comment]
hi @scottwinkler , i think this is due to SetNewComputed which will affect *schema.ResourceData.Get() (i am using v2 of the SDK and didn't notice this problem in my provider), making it return empty map. I think it is safe to use  *schema.ResourceData.GetChange() instead

**Comment by scottwinkler** [comment]
Fix released in 1.7.7
